test/integration/LdpHandlerWithAuth.test.ts

```diff
@@ -53,7 +52,7 @@ describe.each(stores)('An LDP handler with auth using %s', (name, { storeUrn, te
     // Write test resource
     await store.setRepresentation({ path: `${BASE}/permanent.txt` },
       new BasicRepresentation(createReadStream(joinFilePath(__dirname, '../assets/permanent.txt')), 'text/plain'));
-  });
+  }, 20000);
```

Comment:
This should not be added everywhere. There is a single timeout in `jest.config.js` that can be updated. Does this mean you keep running into timeout problems during your tests? I actually have this problem myself from time to time but I assumed it was because I'm working on an 8 year old laptop. If more people run into this we should definitely increase the timeout again.

Reply:
Seemed to only be a problem in some `beforeAll()`s from some integration tests. Will increase in jest.config.js.